app/backplane/sensor_module/src/c_sensing_tenant.cpp

```diff
@@ -85,9 +85,35 @@ void CSensingTenant::Run() {
 
     data.Temperature.Temperature = thermometer.GetSensorValueFloat(SENSOR_CHAN_AMBIENT_TEMP);
 
-    detectionHandler.HandleData(uptime, data, sensor_states);
     // If we can't send immediately, drop the packet
     // we're gonna sleep then give it new data anywas
     dataToBroadcast.Send(data, K_NO_WAIT);
-    dataToLog.Send(data, K_NO_WAIT);
+    sendDownlinkData(data);
+
+    detectionHandler.HandleData(uptime, data, sensor_states);
+    if (detectionHandler.FlightOccurring()) {
+        dataToLog.Send(data, K_NO_WAIT);
+    }
+}
+
+void CSensingTenant::sendDownlinkData(const NTypes::SensorData& data) {
+    NTypes::LoRaBroadcastSensorData downlinkData{
+        .Barometer = {
+            .Pressure = static_cast<int16_t>(data.PrimaryBarometer.Pressure),
+            .Temperature = static_cast<int16_t>(data.PrimaryBarometer.Temperature),
+        },
+        .Acceleration = {
+            .X = static_cast<int16_t>(CSensorDevice::ToMilliUnits(data.Acceleration.X)),
+            .Y = static_cast<int16_t>(CSensorDevice::ToMilliUnits(data.Acceleration.Y)),
+            .Z = static_cast<int16_t>(CSensorDevice::ToMilliUnits(data.Acceleration.Z)),
+        },
+        .Gyroscope = {
```

Comment:
Now that we have the technology, prolly oughta do the scaling to gyro too. 
![image](https://github.com/user-attachments/assets/4544e0fe-b970-4c79-83e0-43ff7afa69d0)

our rotation in radians/s is quite small according to sims - max of 0.325 during boost. Almost def wanna record higher bc that doesnt include snatch (where grim experienced 20 rad/s) or hitting the ground where grim experienced |40| rad/s

but we're losing precision here as well

Reply:
😴